tests/app/dao/test_monthly_billing.py

```diff
@@ -136,3 +159,34 @@ def test_get_service_id(notify_db_session):
                                                                end_date=datetime(2017, 7, 16))
     expected_services = [service_1.id, service_2.id]
     assert sorted([x.service_id for x in services]) == sorted(expected_services)
+
+
+def test_get_monthly_billing_entry_filters_by_service(notify_db, notify_db_session):
+    service_1 = create_service(service_name="Service One")
+    service_2 = create_service(service_name="Service Two")
+    now = datetime.utcnow()
+
+    create_sample_monthly_billing_entry(
+        notify_db,
+        notify_db_session,
+        service_1.id,
+        'sms',
+        [],
+        now,
+        now + timedelta(days=30)
+    )
+
+    create_sample_monthly_billing_entry(
+        notify_db,
+        notify_db_session,
+        service_2.id,
+        'sms',
+        [],
+        now,
+        now + timedelta(days=30)
+    )
```

Comment:
I wonder if this can be made clearer by adding the key to describe the value when passing the arg in? 

Also perhaps the `create_sample_monthly_billing_entry` can have defaults set as the only difference between the 2 entries is the service `id`